.delivery/build-cookbook/recipes/default.rb

```diff
@@ -46,5 +46,6 @@
 ].each do |v|
   execute "umount #{v}" do
     returns [0,1]
+    ignore_failure true
```

Comment:
When we switch to new job dispatch, we won't have these mount points existing on any nodes.